src/_includes/footer.html

```diff
@@ -7,6 +7,12 @@
         &nbsp;&nbsp;|&nbsp;&nbsp;
         <a href="https://docs.google.com/document/d/1jA8Qc8WZMhmaqdjd8Ng8rY4iMI073xCfLNt_DSP72TQ/edit" rel="noopener"
             target="_blank">End User Agreement</a>
-        <div class="copyright-container copyright-white-text">Copyright &copy; 2017 Geotab</div>
+        <div class="copyright-container copyright-white-text">Copyright &copy; 2022 Geotab</div>
+    <script>
```

Comment:
Copywrite was set to 2017 by request of legal department years ago. I think if we want to change this we should inlist the advise of the legal team.